helm/dex-app/values.yaml

```diff
@@ -96,6 +98,10 @@ managementCluster:
 # Giant Swarm properties for workload clusters
 isWorkloadCluster: false
 
+# Enable the deployment of the dex k8s authenticator together with dex
+# Please note that this app is deprecated and will be removed in the future
```

Comment:
If we say that Dex k8s Authenticator is deprecated, users may be tempted to set the flag value to false and not deploy the authenticator.
Are there consequences to that action? Would Dex still work of the user keeps the authenticator disabled and installs Dex by following the instructions in the docs (which reference the authenticator)?

Reply:
Yes, it would work just fine. We have already completely removed it from our own installations already and there were no issues.